index.js

```diff
@@ -58,9 +69,14 @@ module.exports = ({negotiateProtocol}) => {
 
 			return {
 				async compile() {
+					if (relativeCompile) {
+						await compileTypeScript(protocol.projectDir);
+					}
+
 					return {
 						extensions: extensions.slice(),
-						rewritePaths: rewritePaths.slice()
+						rewritePaths: rewritePaths.slice(),
+						compile: Boolean(relativeCompile)
```

Comment:
```suggestion
						compile: relativeCompile
```

It's already a boolean.